### DISCUSSION TITLE
The command '/bin/sh -c pip install -r requirements_dev.txt' returned a non-zero code: 137

### DISCUSSION THREAD

**Discussion by jessicana** [saleor/saleor#9607] [Deployment]
I am trying to install saleor in the root directory. I followed the documentation and I run:

```
git clone https://github.com/saleor/saleor-platform.git --recursive --jobs 3
cd saleor-platform
docker-compose build
```
I got this error:
```
Killed
The command '/bin/sh -c pip install -r requirements_dev.txt' returned a non-zero code: 137
ERROR: Service 'api' failed to build : Build failed
```
How can I fix that?

![image](https://user-images.githubusercontent.com/7415824/164944078-cef1f68f-ce5b-4637-9913-3bd3a956bc77.png)

**Comment by karolkielecki** [comment]
Hi @jessicana, 
Thanks for reporting. I've passed it on to the team. If there is any news, I'll let you know.

**Reply by muctarr-rahim** [reply]
Hi @karolkielecki, any news on this error?

**Reply by karolkielecki** [reply]
Hi @muctarr-rahim, we were not able to reproduce it. We've checked that a few times and haven't received that error. Could you please check the container's memory? Perhaps you need to increase the memory limit.

**Reply by patrys** [reply]
Error code typically means the host machine terminated the process because of memory pressure (ie. computer ran out of memory). I suggest that you build the Docker image locally and then push it to the intended production node, and not attempt to build images on the target system.
